astroquery/eso/core.py

```diff
@@ -701,8 +701,12 @@ def retrieve_data(self, datasets, continuation=False, destination=None,
         if not all(valid_datasets):
             invalid_datasets = [ds for ds, v in zip(datasets_to_download,
                                                     valid_datasets) if not v]
-            raise ValueError("The following data sets were not found on the "
-                             "ESO servers: {0}".format(invalid_datasets))
+            if verbose:
```

Comment:
these changes seem to be unrelated to this PR, please remove the commit.

Reply:
Sorry!